### PR TITLE
feat: add ipfs generate key command

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/orb-cli/ipfskeygencmd/keygen.go [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+	"github.com/spf13/cobra"
+	cmdutils "github.com/trustbloc/edge-core/pkg/utils/cmd"
+	tlsutils "github.com/trustbloc/edge-core/pkg/utils/tls"
+
+	"github.com/trustbloc/orb/cmd/orb-cli/common"
+)
+
+const (
+	ipfsURLFlagName  = "ipfs-url"
+	ipfsURLFlagUsage = "Ipfs url." +
```
Comment (bstasyszyn) [inline cmd/orb-cli/ipfskeygencmd/keygen.go:30]: Should be all caps - IPFS.

File: cmd/orb-cli/ipfskeygencmd/keygen.go [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+	createFlags(cmd)
+
+	return cmd
+}
+
+func keyGenCmd() *cobra.Command { //nolint: funlen,gocyclo,cyclop
+	return &cobra.Command{
+		Use:   "key-gen",
+		Short: "generate ipfs key",
```
Comment (bstasyszyn) [inline cmd/orb-cli/ipfskeygencmd/keygen.go:68]: IPFS

**Review by bstasyszyn — APPROVED** [review]

File: cmd/orb-cli/ipfskeygencmd/keygen.go (hunk 2 above)
Comment (fqutishat) [reply cmd/orb-cli/ipfskeygencmd/keygen.go]: done

File: cmd/orb-cli/ipfskeygencmd/keygen.go (hunk 1 above)
Comment (fqutishat) [reply cmd/orb-cli/ipfskeygencmd/keygen.go]: done

**Review by sandrask — APPROVED** [review]

**Pull request merged**